conans/tools.py

```diff
@@ -121,7 +121,18 @@ def environment_append(env_vars):
         os.environ.update(old_env)
 
 
-def build_sln_command(settings, sln_path, targets=None, upgrade_project=True):
+def msvc_command(settings, sln_path, targets=None, upgrade_project=True, build_type=None,
```

Comment:
Hard to know what it does reading the name, specially having the build_sln_command...